app/models/concerns/waste_carriers_engine/can_have_registration_attributes.rb

```diff
@@ -157,6 +157,12 @@ def update_last_modified
       def declaration_confirmed?
         declaration == 1
       end
+
+      def unpaid_balance?
+        return true if finance_details.presence&.balance&.positive?
```

Comment:
Just a question: the previous version of the code always assumed a finance_details, if present, should always have a `balance` present that has a numeric value/object. Is this not the case any more?

Reply:
It should have a `balance` (although MongoDB is less strict about this sort of thing). But Rubocop kicked off when I had `finance_details.presence&.balance.positive?` – I think it doesn't like to have `.` calls chained after `&.` calls.

Reply:
Thanks for the explanation! :)